paper.md

```diff
@@ -0,0 +1,74 @@
+---
+title: 'CLOVER: Community-scale sustainable energy system modelling'
+tags:
+ - Python
+ - energy access
+ - minigrid
+ - renewable energy
+ - sustainable development
+authors:
+ - name: Philip Sandwell
```

Comment:
This YAML doesn't look indented correctly:
```yaml
authors
 - name: Philip Sandwell
 orcid: ...
```
I'll fix this and open a PR with updated YAML :smile:

Reply:
Thanks, I merged your pull request to amend this